src/QueryAutomater.py

```diff
@@ -0,0 +1,19 @@
+import requests
+
+def query_automater():
+    request_data = requests.get('http://www.mocky.io/v2/5e4ddd0a2f00002b0016a28a')
+    data = request_data.json()
+
+    query = "SELECT * WHERE {GRAPH ?g {?s ?p ?o}"   # basic n-quads query format
```

Comment:
Let's have another method called `get_query(query_type): string`, which accepts a query_type enum and return a base query(as documented in the `queries` folder).

`query_type` enum will have four options `{Article, Website, Organization, ..}`